optuna/samplers/_tpe/sampler.py

```diff
@@ -143,9 +162,21 @@ def __init__(
         self._gamma = gamma
         self._weights = weights
 
+        self._logger = get_logger(__name__)
```

Comment:
Can you define the logger at module level? Sorry it might not be clear but it's a style adopted in Optuna. You can take a look at `_cmaes.py` for how it's done https://github.com/optuna/optuna/blob/ee08edccfb6b0c204ff7cd2c6a67baaa5166635c/optuna/samplers/_cmaes.py#L21.

Reply:
Thank you for pointing out! I was having a problem with testing around logger.

Reply:
It seems that _logger defined at module level is not used in the _log_independent_sampling in cma. `self._logger` is used there. Could you elaborate on why this is the case?

Reply:
You're right. I think it's unintentional.

Reply:
done and passed CI